design/keycloak.x/configuration.md

```diff
@@ -373,11 +381,33 @@ By default only supported features are enabled. In addition there are some suppo
 To enable non-supported features you can either enable individual features or change the feature level with `--features=<level>`. 
 For example `--features=preview` would enable all preview and deprecated features.
 
-## Show Configuration
+## Commands
+
+### `start`
+
+This command starts a new server instance.
+
+### `show-config`
 
 Keycloak can read configuration properties from multiple sources (environment variables, CLI, and property files). In addition to 
 that, the server's configuration can also be persisted when running the `config` command.
 
+With that in mind, the `show-config` command is provided to check the properties that are set into the server image as well as those
+that were set at runtime when starting the server.
+
+      kc.[sh|bat] show-config
+
+The example above is going to print only the configuration set into the server image without start the server.
+
+The `show-config` command aims to help troubleshooting and investigations in case the server is not behaving or not configured as expected.
+
+The `start` method also supports using this command as a sub-command:
+
+      kc.[sh|bat] start show-config
```

Comment:
Do we really need show-config as a sub-command?

Reply:
We have a similar command in our tooling (ci/cd) and it helped us a lot. I would vote for it. It can also be used to automate tests agains the config.

Reply:
It's confusing to have two commands being used, and we don't have the concept of a "sub-command" anywhere else. I'd say if we want the ability to show config and start at the same time it should be done with supporting multiple commands, like "kc.sh config start", which should then be possible for everything. Or, we can have an arg to start command "kc.sh start --show-config".

Reply:
Agree. Will change that on both sides (dist and here).

@bs-matil good to know.